src/panels/lovelace/components/hui-theme-select-editor.ts

```diff
@@ -0,0 +1,71 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import "@polymer/paper-button/paper-button";
+import { TemplateResult } from "lit-html";
+
+import { HomeAssistant } from "../../../types";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+
+export class HuiThemeSelectionEditor extends hassLocalizeLitMixin(LitElement) {
+  public value?: string;
+  protected hass?: HomeAssistant;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      value: {},
+    };
+  }
+
+  protected render(): TemplateResult {
+    const themes = ["Backend-selected", "default"].concat(
+      Object.keys(this.hass!.themes.themes).sort()
+    );
+
+    return html`
+      ${this.renderStyle()}
+      <paper-dropdown-menu
+        label="Theme"
+        dynamic-align
+        @value-changed="${this._changed}"
+      >
+        <paper-listbox
+          slot="dropdown-content"
+          .sselected="${this.value || "Backend-selected"}"
+          .attrForSelected="theme"
```

Comment:
This doesn't work, my bad for suggesting otherwise. Make this `attr-for-selected="theme"`